src/qt/overviewpage.cpp

```diff
@@ -176,6 +178,9 @@ void OverviewPage::handleTransactionClicked(const QModelIndex &index)
 
 void OverviewPage::setPrivacy(bool privacy)
 {
+    QSettings settings;
+    settings.setValue("privacy", privacy);
```

Comment:
In commit "Persist "mask values" in gui" (9fde0bae916f73b73fcdbeff6469dba72c4be212)

I don't think this is too important, but I'm not sure if I agree with furszy's comment that this setting needs to be part of `OptionsModel`. It seems nice to make it part of `OptionsModel` but also reasonable to not make it part of `OptionsModel` and just read/write it with `QSettings` directly. 

But probably it would be better to pick one approach or the other. Either don't add the setting to OptionsModel and use QSettings to read and write. Or do add the setting to OptionsModel and use OptionsModel to read and write. Otherwise you wind up causing inconsistent access to the setting across layers that furszy was trying to avoid https://github.com/bitcoin-core/gui/pull/655#pullrequestreview-1103143874